load_easybuild_module.sh

```diff
@@ -75,17 +75,22 @@ else
     if [[ $? -eq 0 ]]; then
         echo_green ">> Module for EasyBuild v${EB_VERSION} found!"
```

Comment:
Probably, we should also set `IGNORE_CACHE=''` here, just to be sure it is an empty string (since we will rely on that in line `111`, when we do the `module ${IGNORE_CACHE} load ...`.

Reply:
Better yet, do it on line `74` so that `''` is the default value for all cases. Then, the second case (where we actually ignore the cache to find the module) is the only exception - and that then overwrites `IGNORE_CACHE='--ignore_cache'`